numba/core/utils.py

```diff
@@ -472,7 +472,7 @@ def unify_function_types(numba_types):
     return (dtype,) * len(numba_types)
 
 
-def unified_function_type(numba_types):
+def unified_function_type(numba_types, require_precise=True):
```

Comment:
This kwarg addition should be documented. What's it's purpose and the effect of having it not set?

Reply:
Updated unified_function_type docstring.